src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -342,6 +342,17 @@ ANALYTICS_IDS = {
     'APPCUES_KEY': '{{ APPCUES_KEY }}'
 }
 
+GOOGLE_OAUTH_CONFIG = {
+    "web": {
+        "client_id": '{{ GOOGLE_OAUTH_CLIENT_ID }}',
+        "project_id": '{{ GOOGLE_OAUTH_PROJECT_ID }}',
+        "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+        "token_uri": "https://oauth2.googleapis.com/token",
+        "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+        "client_secret": '{{ GOOGLE_OAUTH_CLIENT_SECRET }}'
+    }
+}
+
```

Comment:
Is google sheets functionality optional for self-hosters? If so, we might think about putting this block in a conditional, depending on if the google oauth values are present

Reply:
we haven't really spoken about self hosters yet